app/controllers/users/registrations_controller.rb

```diff
@@ -50,6 +58,16 @@ def create
       end
     end
 
+    def phone_validation
+      user = User.find_by_phone(params[:phone])
```

Comment:
je pense que ca peut lever un active record not found, et il y aura un prob en cas de nil.